core/src/main/java/org/infinispan/interceptors/CacheMgmtInterceptor.java

```diff
@@ -48,7 +48,7 @@
    private final LongAdder hits = new LongAdder();
    private final LongAdder misses = new LongAdder();
    private final LongAdder stores = new LongAdder();
-   private final LongAdder evictions = new LongAdder();
+   private final AtomicLong evictions = new AtomicLong(0);
```

Comment:
Is there a reason for changing this?  LongAdder tends to perform better under concurrent access.